frontend/src/components/ProjectDeploymentConfigPanel.vue

```diff
@@ -130,14 +140,15 @@ import {
   LabelSelectorRequirement,
 } from "../types";
 import DeploymentConfigTool, { DeploymentMatrix } from "./DeploymentConfigTool";
-import { cloneDeep } from "lodash-es";
+import { cloneDeep, isEqual } from "lodash-es";
 import { useI18n } from "vue-i18n";
-import { NPopover } from "naive-ui";
+import { NPopover, useDialog } from "naive-ui";
 import { generateDefaultSchedule, validateDeploymentConfig } from "../utils";
 
 type LocalState = {
+  deployment: DeploymentConfig | undefined;
+  backup: DeploymentConfig | undefined;
```

Comment:
backup is an unfortunate name since we have a backup concept.

The convention is to use originalXXX for this type of check.

Reply:
That's unfortunate indeed.
Updated